packages/squiggle-lang/__tests__/TS/PointSet_test.ts

```diff
@@ -0,0 +1,49 @@
+import { errorValueToString } from "../../src/js/index";
+import { testRun, failDefault, expectErrorToBeBounded } from "./TestHelpers";
+import * as fc from "fast-check";
+
+describe("Mean of mixture is weighted average of means", () => {
+  test("mx(beta(a,b), lognormal(m,s), [x,y])", () => {
+    fc.assert(
+      fc.property(
+        fc.float({ min: 1e-1 }), // alpha
+        fc.float({ min: 1 }), // beta
+        fc.float(), // mu
+        fc.float({ min: 1e-1 }), // sigma
+        fc.float({ min: 1e-7 }),
+        fc.float({ min: 1e-7 }),
+        (a, b, m, s, x, y) => {
+          let squiggleString = `mean(mx(beta(${a},${b}), lognormal(${m},${s}), [${x}, ${y}]))`;
+          let res = testRun(squiggleString);
```

Comment:
Can ``testRun`` just throw errors if it fails? (Recommended by Sam)

Reply:
It returns squiggle error objects when squiggle fails in the current functionality. The version of `testRun` in `JS_test.ts` (which I did not touch) does a little more work. 

Are you suggesting `testRun` throws to the js/ts level when there's a squiggle error?

Reply:
> Are you suggesting testRun throws to the js/ts level when there's a squiggle error?

Correct

Reply:
hm but I like the idea of being able to check errors paths/states/objects are what we expect them to be. (not that I'm in effect doing that very much, but I thought I'd have to more)